packages/expo-task-manager/android/build.gradle

```diff
@@ -73,5 +80,12 @@ dependencies {
   unimodule "unimodules-constants-interface"
   unimodule "expo-app-loader-provider"
 
+  implementation "com.facebook.react:react-native:+"
```

Comment:
I'm wondering whether we should move our code that depends on React Native to separate package or not. The idea of unimodules architecture was to provide modules that don't depend on the framework that other users may use. For example, adding support for Flutter would require some more work as we don't want to require Flutter's users to depend on RN. What about moving app loading implementation (depending on RN) to the adapter?